### PR TITLE
Implement ability to use built-in datadog logging support

### PR DESCRIPTION
**Pull request by s-vitaliy**
Part of #61

## Scope

Implemented:
- Logging to datadog using push mechanism with sl4j
- Logging to console with simple format

Additional changes:
- To be able to enrich the sl4j logs created directly **and** the zio logs we should do the follwoing:
   - append properties to zio logger
   - addend properties to java MDC
To be able to add that metadata to both log sources in the simple manner, I created the `Enircher` interface and it's implementations that should be used in main.scala in the logging bootstrap process.

## Checklist

- [x] GitHub issue exists for this change.
- [ ] Unit tests added and they pass.
- [ ] Line Coverage is at least 80%.
- [ ] Review requested on `latest` commit.

### CODE REVIEW

File: plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala [hunk 1]
```diff
@@ -7,18 +7,31 @@ import com.sneaksanddata.arcane.framework.models.DataRow
...
 import zio.logging.backend.SLF4J
-import zio.{Chunk, ZIO, ZIOAppDefault, ZLayer}
+import zio.{ZIO, ZIOAppDefault, ZLayer}
 
 object main extends ZIOAppDefault {
 
-  override val bootstrap: ZLayer[Any, Nothing, Unit] = SLF4J.slf4j(LogFormat.colored)
+  private val loggingProprieties = Enricher("Application", "Arcane.Stream.Scala")
+    ++ Enricher("App", "Arcane.Stream.Scala")
+    ++ Enricher("App", "Arcane.Stream.Scala")
```
Comment (george-zubrienko) [inline plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala:25]: Duplicated

File: plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala [hunk 2]
```diff
@@ -7,18 +7,31 @@ import com.sneaksanddata.arcane.framework.models.DataRow
...
+    ++ Enricher.fromEnvironment("APPLICATION_VERSION", "local-development")
```
Comment (george-zubrienko) [inline plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala:26]: 0.0.0?

**Review by george-zubrienko — APPROVED** [review]

File: plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala (hunk 2 above)
Comment (s-vitaliy) [reply plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala]: Fixed

File: plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala (hunk 1 above)
Comment (s-vitaliy) [reply plugin/arcane-stream-sqlserver-change-tracking/src/main/scala/main.scala]: Fixed

**Pull request merged**